packages/appium/test/fixtures/flattened-schema.js

```diff
@@ -11,14 +11,6 @@ export default [
       ref: 'appium.json#/properties/server/properties/address',
     },
     schema: {
-      anyOf: [
```

Comment:
I am not happy about dropping the whole argument validation just because of a bug in typing.
At least the validation should be added elsewhere instead of removing it completely

Reply:
I'm not happy about it either.  I'll create an issue to deal w/ this